src/dao/hooks/useDAOProposals.tsx

```diff
@@ -164,7 +163,7 @@ export function useDAOProposals(
             const proxyAdmin = solidityAddressToAccountIdString(upgradeProposalData.proxyAdmin);
             const proxyLogic = (await DexService.fetchContractId(upgradeProposalData.proxyLogic)).toString();
             parsedData = { ...upgradeProposalData, proxyAdmin, proxyLogic, currentLogic };
-            isAdminApproved = latestAdminLog?.newAdmin?.toLocaleLowerCase() === safeEVMAddress.toLocaleLowerCase();
+            isAdminApproved = latestAdminLog?.newAdmin?.toLocaleLowerCase() === safeAccountId.toLocaleLowerCase();
```

Comment:
can we rename `safeAccountId `to `safeEVMAddress ` address since its a evm address.

Reply:
Done.